### DISCUSSION TITLE
MOSH Color support

### DISCUSSION THREAD

**Discussion by agluck91** [blinksh/blink#1756] [Q&A] [answered]
I notice when using mosh that is does not have full color support, My starship config looked like junk when I logged in. Is there something I have to do to add full color support? I saw on mosh’s git that they release full color support like 3 years ago, I am just not sure how to enable it.

**Comment by carloscabanero** [comment, marked as answer]
Although the PR for TrueColor had been there for a while, it made it to an official release just a few months ago (https://github.com/mobile-shell/mosh/releases/tag/mosh-1.4.0), and probably your pm may not have it yet. You may need to compile from master or grab a binary and install it on the server. That is what we do on Blink Build to offer Mosh out of the box.

**Reply by agluck91** [reply]
Thank you Carlos, you are the bomb. I will get that added to my base image!!

**Reply by agluck91** [reply]
For anyone that finds this later, I did want to say this is the solution. 

Steps I did: 

Grab the latest release from their github page
untar it on your server

run 
`./configure`
`make`
`make check`
`sudo make install`

then verify that your are now running the new version type `mosh —version`. As of writing this it was 1.4.0. You need to have 1.4.0 or higher to get the color support.
